staging/src/k8s.io/pod-security-admission/test/run.go

```diff
@@ -140,15 +183,12 @@ func Run(t *testing.T, opts Options) {
 	if err != nil {
 		t.Fatalf("invalid checks: %v", err)
 	}
-	maxMinor, err := maxMinorVersionToTest(opts.Checks)
-	if err != nil {
-		t.Fatalf("invalid checks: %v", err)
-	}
 
-	for _, level := range []api.Level{api.LevelBaseline, api.LevelRestricted} {
-		for minor := 0; minor <= maxMinor; minor++ {
-			version := api.MajorMinorVersion(1, minor)
+	versionsToTest := computeVersionsToTest(t, opts.Checks)
```

Comment:
optional: consider exposing this approach in options (e.g. `opts.RunAllVersions`)

Reply:
added a TODO to consider that